content/recipes/interoperability/ontology-operations-tools.md

```diff
@@ -273,410 +55,138 @@ The table below is an overview of ontology strategies tools identified. Details
+|Tool|Description|License|Topics|Resource Type|How to use|
+|---|--|--|--|--|--|
+|[ZOOMA](https://www.ebi.ac.uk/spot/zooma/)|A tool for mapping free text annotations to ontology term based on a curated repository of annotation knowledge|[EMBL-EBI Terms of Use](https://www.ebi.ac.uk/about/terms-of-use/)|Ontology and terminology,<br>Systems biology,<br>Data identity and mapping|Web application,<br> API|[ZOOMA-Getting started](https://www.ebi.ac.uk/spot/zooma/docs/search)|
+|[NCBI BioPortal Annotator](https://bioportal.bioontology.org/annotatorplus)|Get annotations for biomedical text with classes from the ontologies.|[BioPortal Terms of Use](https://www.bioontology.org/terms/)|Ontology and terminology,<br>Systems biology,<br>Data identity and mapping|Web application,<br> API|[BioPortal help](https://bioportal.bioontology.org/help?pop=true#Annotator_Tab)|
 |[BioBert](https://github.com/dmis-lab/biobert)|A biomedical language representation model designed for biomedical text mining tasks such as biomedical named entity recognition, relation extraction, question answering.|[Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)|text mining,<br> named-entity recognition,<br> natural language processing|Python|
 |[Termite](https://www.scibite.com/platform/termite/)|Semantic enrichment to unlock the value of unstructured text and simplify the identification of new potential biomarker leads from scientific text.|Commercial license|Ontology and terminology|
 |[PoolParty Semantic Suite](https://semantic-web.com/poolparty-semantic-suite/)|Automate the handling of heterogeneous metadata systems and the creation of enterprise knowledge graphs.design knowledge graphs at your own pace and with speed. Create your own ontologies and custom schemes by reusing already existing ontologies such as FOAF, FIBO, schema.org and CHEBI, among others. Apply them to your existing taxonomies with ease.|Commercial license|Content enrichment,<br>Data integration|
 |[OntoMaton](https://github.com/ISA-tools/OntoMaton)| A tool facilitating ontology search and tagging functionalities within Google Spreadsheets.|[CPAL license](https://opensource.org/licenses/CPAL-1.0)||Google Add-ons|
 |[Prodigy](https://prodi.gy/)|A modern annotation tool for creating training and evaluation data for machine learning models. You can also use Prodigy to help you inspect and clean your data, do error analysis and develop rule-based systems to use in combination with your statistical models.|Commercial license|Data annotation|Python, Web application,API|
 |[OntoText](https://www.ontotext.com/products/ontotext-platform/)|Connect and publish complex enterprise knowledge with standard-compliant semantic graph database;Customize and apply analytics to link documents to graphs, extract new facts, classify and recommend content...|Commercial license|
```

Comment:
Replace semicolon with full stop and space

Reply:
thanks for pointing that out. now fixed/